cms/src/components/content/EditContentSeo.vue

```diff
@@ -0,0 +1,33 @@
+<script setup lang="ts">
```

Comment:
Do we really need a separate component here? Could it not be included in EditContentGeneral (and perhaps rename EditContentGeneral to EditContentTitleSummary)?